pkg/controller/linstorcontrollerset/linstorcontrollerset_controller.go

```diff
@@ -178,6 +178,13 @@ func (r *ReconcileLinstorControllerSet) Reconcile(request reconcile.Request) (re
 		return reconcile.Result{}, fmt.Errorf("CS Reconcile: missing required parameter dbConnectionURL: outdated schema")
 	}
 
+	if pcs.Spec.Ssl != nil {
+		// 2020-05-12 FIXME: Teach LINSTOR how to read these parameters from controller config. Currently these values are added directly to the database on initial migration.
```

Comment:
I wouldn't write FIXME here because this is nothing that can be fixed in this project. Just write "LINSTOR cannot currently ...".